examples/autocomplete.amp.html

```diff
@@ -591,7 +591,7 @@ <h2>Templates in search context</h2>
                         ] }
                     </script>
                     <template type="amp-mustache" id="amp-template-default">
-                        <div class="city-item" data-value="{{value}}">
+                        <div class="city-item" value="{{value}}">
```

Comment:
The HTML Specification doesn't include the usage of the `value` attribute on `HTMLDivElement`s. Are we sure we want to allow arbitrary `HTMLElement`s to use potentially spec incompatible attributes?

Reply:
Closing the PR and updated the issue. Thanks!